### PR TITLE
Race fix

### PR DESCRIPTION
**Pull request by scribe**
Fairly strange fix, race condition caused by popups and specific timing, and fixing the fallout from that because the code is coupled.

### CODE REVIEW

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/AlertService.java [hunk 1]
```diff
@@ -48,8 +43,6 @@ public AlertService(final ResourceBundle resourceBundle) {
 
 
     private void showAlertInternal(final String message, final String title, final Alert.AlertType alertType, final Window window) {
-        Platform.runLater(() -> {
```
Comment (shabtaisharon) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/AlertService.java]: removing that will still show the popup?

Comment (scribe) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/AlertService.java]: Yeah, it still shows, but now it doesn't lock the UI.

Comment (shabtaisharon) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/AlertService.java]: ohh... right this is blocking call

Comment (rpmoore) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/AlertService.java]: It's not a blocking call.

Comment (shabtaisharon) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/AlertService.java]: UI blocking

Comment (rpmoore) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/AlertService.java]: Can't this method be called from a non-javafx thread?

**Pull request merged**